### PR TITLE
Added SAMM meta model class for units

### PR DESCRIPTION
**Pull request by Hanna-Shalamitskaya-EPAM**
## Description

Changes:
- Added SAMM meta model module;
- Created SammUnitsGraph class for getting access to the parameter units;
- Added unit tests for the SammUnitsGraph class.

## Type of change

Please delete options that are not relevant.

- [X] New feature (non-breaking change which adds functionality)

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: core/esmf-aspect-meta-model-python/esmf_aspect_meta_model_python/samm_meta_model.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
+#  Copyright (c) 2023 Robert Bosch Manufacturing Solutions GmbH
```
[the remark this answers is not in the document]
Comment (Hanna-Shalamitskaya-EPAM) [reply core/esmf-aspect-meta-model-python/esmf_aspect_meta_model_python/samm_meta_model.py:1]: No, should be the current year.
Thanks for for carefully checking

[the remark this answers is not in the document]
Comment (Hanna-Shalamitskaya-EPAM) [reply core/esmf-aspect-meta-model-python/esmf_aspect_meta_model_python/samm_meta_model.py]: Fixed

File: core/esmf-aspect-meta-model-python/README.md [hunk 2]
```diff
@@ -63,3 +63,29 @@ This script can be executed with
 poetry run download-samm-branch
 ```
 to download and start working with the Aspect Model Loader.
+
+## SAMM meta model
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/README.md:67]:
```suggestion
## Semantic Aspect Meta Model
```

File: core/esmf-aspect-meta-model-python/README.md [hunk 3]
```diff
@@ -63,3 +63,29 @@ This script can be executed with
...
+
+To be able to use "SAMM meta model" classes you need to download "SAMM Aspect Meta Model for development" ([Set Up SAMM Aspect Meta Model for development](#Set Up SAMM Aspect Meta Model for development)).
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/README.md:69]:
```suggestion
To be able to use SAMM meta model classes you need to download the corresponding files. Details are described in [Set up SAMM Aspect Meta Model for development](#set-up-samm-aspect-meta-model-for-development).
```

File: core/esmf-aspect-meta-model-python/README.md [hunk 4]
```diff
@@ -63,3 +63,29 @@ This script can be executed with
...
+
+This module contains classes for working with SAMM data.
```
Comment (atextor) [inline core/esmf-aspect-meta-model-python/README.md:71]:
```suggestion
This module contains classes for working with Aspect data.
```

**Review by atextor — CHANGES_REQUESTED** [review]

**Review by atextor — APPROVED** [review]

**Pull request merged**